testing/python/fixtures.py

```diff
@@ -1092,6 +1092,26 @@ def test_1(arg):
         reprec = pytester.inline_run()
         reprec.assertoutcome(passed=2)
 
+    @pytest.mark.parametrize("name", ["path", "fspath", "module"])
+    def test_session_scoped_unavailable_attributes(self, pytester, name):
+        pytester.makepyfile(
+            f"""
+            import pytest
```

Comment:
i really wish we could do tests like this one as unittest

Reply:
IMHO that'd require fixtures to be importable/scoped in some way, rather than essentially being global state.